console/src/components/Common/AceEditor/utils.ts

```diff
@@ -1,3 +1,5 @@
+// eslint-disable-file import/no-extraneous-dependencies
```

Comment:
So ace-editor installs `ace-builds` along with itself.
I didn't add it separately as a dependency because I am not aware if there are any versioning incompatibility issues.
If there's surety that there aren't any, it can be added to deps.